src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const employeeList = [...document.querySelectorAll('li')];
+const mainList = document.querySelector('ul');
+
+function salarySort(list) {
+  list.sort((a, b) => (+b.dataset.salary.slice(1).split(',').join(''))
+   - (+a.dataset.salary.slice(1).split(',').join('')));
```

Comment:
mb you already have some function that get info from dataset?

Reply:
Hi, I see you haven't fixed previous comments, maybe you did not push your changes?
If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review 🙂